### PR TITLE
Additional changes to FpsProcessor, fps instrument and uxperf result processor

### PR DESCRIPTION
**Pull request by johnrchrdsn**
Includes :
- Fix for FpsProcessor logic when data for filtered_vsyncs_to_compose is empty
- Adds dumpsys_period parameter for controlling the frequency of dumpsys in the fps instrument
- New logging calls and exception handling in uxperf result processor.

### CODE REVIEW

File: wlauto/utils/fps.py [hunk 1]
```diff
@@ -19,8 +19,6 @@ class FpsProcessor(object):
     Provide common object for processing surfaceFlinger output for frame
     statistics.
 
-    This processor adds four metrics to the results:
```
Comment (ep1cman) [inline wlauto/utils/fps.py]: Why was this line removed?

Comment (johnrchrdsn) [inline wlauto/utils/fps.py]: Originally the code added the metrics to the context but that has now changed. This comment is therefore a bit misleading.

Comment (ep1cman) [inline wlauto/utils/fps.py]: Rather than removing it I would change it to say that it returns the values below

File: wlauto/result_processors/uxperf.py [hunk 2]
```diff
@@ -86,12 +87,15 @@ def export_iteration_result(self, result, context):
         logfile = os.path.join(context.output_directory, 'logcat.log')
         framelog = os.path.join(context.output_directory, 'frames.csv')
 
+        self.logger.info('Parsing logcat.log for UX_PERF markers')
```
Comment (ep1cman) [inline wlauto/result_processors/uxperf.py]: This should also go to the debug log
